src/js/models/layer.js

```diff
@@ -134,12 +134,12 @@ define(function (require, exports, module) {
         /**
          * @type {Immutable.List.<Shadow>}
          */
```

Comment:
there are about 10 Immutable.List attributes defined with null default value after a quick search, shall I refactor them to keep consistency or wait until our major release?